pkg/cmd/root/help_topic_test.go

```diff
@@ -4,76 +4,83 @@ import (
 	"testing"
 
 	"github.com/cli/cli/v2/pkg/iostreams"
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
-func TestNewHelpTopic(t *testing.T) {
+func findHelpTopic(t *testing.T, name string) helpTopic {
```

Comment:
There are some things we could do about this search but I don't think it's really a big deal. It sort of depends on what we're really trying to test here. It seems like maybe the test in this file is about how the `NewCmdHelpTopic` function uses a `helpTopic`, rather than `environment` specifically.

In that case we could just create a `helpTopic` directly in the test. This would also be nice because we could inject a `helpTopic` that overrides the `helpFunc` default. I haven't done this yet just because I already had a few things I wanted to discuss.